### PR TITLE
feat: Hash text into sha512

### PR DESCRIPTION
**Pull request by mgeerse**

### CODE REVIEW

File: functions/sha512/1.0/function.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "description": "Description",
+  "label": "Sha 512",
+  "category": "Misc",
+  "icon": "CreateIcon",
```
Comment (thomasvanophem) [inline functions/sha512/1.0/function.json:5]: I think we need a new icon for this function

File: functions/sha512/1.0/function.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "description": "Description",
```
Comment (thomasvanophem) [inline functions/sha512/1.0/function.json:2]: You could add `"expression": true` to indicate this is an expression function

File: functions/sha512/1.0/function.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  "label": "Sha 512",
+  "category": "Misc",
+  "icon": "CreateIcon",
+  "options": [
+    {
+      "label": "Input",
+      "name": "input",
+      "meta": {
+        "type": "Text"
```
Comment (thomasvanophem) [inline functions/sha512/1.0/function.json:11]: We could make `input` required

**Comment by thomasvanophem** [comment]
Could you write some tests for this function?

**Review by thomasvanophem — CHANGES_REQUESTED** [review]

File: functions/sha512/1.0/function.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    {
+      "name": "output",
+      "label": "As",
+      "meta": {
+        "type": "Output",
+        "output": {
+          "type": "Text"
+        }
+      }
+    }
```
Comment (thomasvanophem) [inline functions/sha512/1.0/function.json:30]: I think this option should be required as well

**Review by thomasvanophem — APPROVED** [review]

File: functions/sha512/1.0/function.json (hunk 4 above)
Comment (mgeerse) [reply functions/sha512/1.0/function.json:30]: Added

**Review by thomasvanophem — APPROVED** [review]

**Pull request merged**